spec/src/main/asciidoc/jakarta-stl.adoc

```diff
@@ -1,5 +1,8925 @@
+|===
+|Font Style |Uses
+|Italic |Emphasis,
+definition of term.
+
+| _Monospace_ |Syntax,
+code examples, attribute names, Java language types, API, enumerated
+attribute values.
+|===
+
+== Acknowledgments
+
+The JavaServer Pages™ Standard Tag Library
+(JSTL) specification is the result of collaborative work involving many
+individuals, all driven by a common goal of designing the best libraries
+possible for the JSP page author community.
+
+We would like to thank all members of the JSR-52
+expert group: Nathan Abramson, Shawn Bayern, Hans Bergsten, Paul
+Bonfanti, Vince Bonfanti, David Brown, Larry Cable, Tim Dawson, Morgan
+Delagrange, Bob Foster, David Geary, Scott Hasse, Hal Hildebrand, Jason
+Hunter, Serge Knystautas, Mark Kolb, Wellington Lacerda, Jan Luehe, Geir
+Magnusson Jr., Dan Malks, Craig McClanahan, Richard Morgan, Glenn
+Nielsen, Rickard Oberg, Joseph B. Ottinger, Eduardo Pelegri-Llopart, Sam
+Pullara, Tom Reilly, Brian Robinson, Russ Ryan, Pasi Salminen, Steven
+Sargent, Allan Scott, Virgil Sealy, Magnus Stenman, Gael Stevens, James
+Strachan, Christine Tomlinson, Norbert von Truchsess, Keyton Weissinger,
+Clement Wong, Alex Yiu.
+
+This specification was first initiated by
+Eduardo Pelegri-Llopart. Eduardo's leadership in making the Java
+platform the best technology available for the web layer has been key in
+shaping the vision behind the standard tag library.
+
+Shawn Bayern and Hans Bergsten deserve special
+credit for being actively involved in all design issues of this
+specification. Their vast expertise and commitment to excellence has had
+a profound impact in every single aspect of this specification. Mille
+mercis Shawn et Hans! Don't know how we would have done it without you
+two.
+
+Many thanks to Jan Luehe for taking ownership of
+the internationalization and formatting chapters of this specification
+on short notice, and doing an incredible job.
+
+Special mention to Nathan Abramson for being a
+driving force behind the expression language introduced in JSTL, to
+James Strachan for leading the group in our understanding of XML for
+page authors, and to Craig McClanahan for his help on servlet and J2EE
+platform related issues.
+
+This specification has drawn a lot of its design
+ideas from pioneers in the field of tag libraries. We are grateful to
+the Jakarta project at Apache, as well as other efforts in the industry,
+where projects led by Craig McClanahan (Struts), James Strachan (XTags),
+Morgan Delagrange (DBTags), Tim Dawson (I18N), Glenn Nielsen (many
+utility taglibs), Scott Hasse (JPath), Dmitri Plotnikov (JXPath), Pasi
+Salminen (O&D Struts), have greatly influenced the design of the JSTL
+libraries.
+
+The RI team composed of Shawn Bayern (lead),
+Nathan Abramson, Justyna Horwat, and Jan Luehe has done a wonderful job
+at turning code faster than the specification could be written.
+
+Quality has been in the capable hands of Ryan
+Lubke, lead of the TCK team that also includes Lance Andersen. David
```

Comment:
The TCK team under the JCP, to distinguish is from the current Eclipse TCK team.